netbout-web/src/main/java/com/netbout/dynamo/DyInbox.java

```diff
@@ -67,6 +69,7 @@
 @Loggable(Loggable.DEBUG)
 @ToString(of = "self")
 @EqualsAndHashCode(of = { "counter", "region", "self", "since" })
+@SuppressWarnings({ "PMD.TooManyMethods", "PMD.ExcessiveImports" })
```

Comment:
@komaz puzzle (todo) in code that will be converted into task, just look for `@todo` in code for example